error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/IdentityConversion.java

```diff
@@ -70,7 +70,10 @@ public final class IdentityConversion extends BugChecker implements MethodInvoca
           staticMethod()
               .onClass("reactor.core.publisher.Flux")
               .namedAnyOf("concat", "firstWithSignal", "from", "merge"),
-          staticMethod().onClass("reactor.core.publisher.Mono").namedAnyOf("from", "fromDirect"));
+          staticMethod().onClass("reactor.core.publisher.Mono").namedAnyOf("from", "fromDirect"),
+          staticMethod()
+              .onClass("com.google.errorprone.matchers.Matchers")
```

Comment:
We sort this list lexicographical, so we should put this one after the `"com.google.common"` ones. Moving it up a bit :).